datafusion/physical-expr/src/equivalence/ordering.rs

```diff
@@ -39,7 +39,7 @@ use arrow_schema::SortOptions;
 /// ordering. In this case, we say that these orderings are equivalent.
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Default)]
 pub struct OrderingEquivalenceClass {
-    pub orderings: Vec<LexOrdering>,
+    orderings: Vec<LexOrdering>,
```

Comment:
This is the key change -- making this field non-pub -- the rest of the PR is derived from that and added comments